### PR TITLE
New package: Lime3DS.Lime3DS version 2107

### LINKED ISSUE #148602: [Package Request]: Lime3ds
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Lime3DS/Lime3DS/releases/download/2107/lime3ds-2107-windows-msvc.zip
Publisher: Lime3DS
Package Name:Lime3DS
Description: a popular open-source Nintendo 3DS emulator 
Package Version: Lime3DS 2107
Installer URL: https://github.com/Lime3DS/Lime3DS/releases/download/2107/lime3ds-2107-windows-msvc.zip

### PR DESCRIPTION
**Pull request by Exorcism0666**
﻿Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

* Resolves #148602

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148636)

### CODE REVIEW

File: manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Lime3DS.Lime3DS
+PackageVersion: "2107"
+PackageLocale: en-US
+Publisher: Lime3DS
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
```
Comment (Trenly) [inline manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.locale.en-US.yaml:11]:
```suggestion
PublisherUrl: https://github.com/Lime3DS
PublisherSupportUrl: https://github.com/Lime3DS/Lime3DS/issues
# PrivacyUrl:
# Author:
```

File: manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+PackageName: Lime3DS
+# PackageUrl:
```
Comment (Trenly) [inline manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://github.com/Lime3DS/Lime3DS
```

File: manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Lime3DS.Lime3DS
+PackageVersion: "2107"
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: lime3ds-2107-windows-msvc\lime.exe
+  InstallerUrl: https://github.com/Lime3DS/Lime3DS/releases/download/2107/lime3ds-2107-windows-msvc.zip
+  InstallerSha256: 9212249DA4BEEC252A96567376C118112F19250DA9149B8658C4FD5B1E74FD99
```
Comment (Trenly) [inline manifests/l/Lime3DS/Lime3DS/2107/Lime3DS.Lime3DS.installer.yaml:13]:
```suggestion
  InstallerSha256: 9212249DA4BEEC252A96567376C118112F19250DA9149B8658C4FD5B1E74FD99
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**